src/lib/containers/widgets/RadioGroup.tsx

```diff
@@ -1,27 +1,27 @@
 import React, { useState } from 'react'
 import { uniqueId as _uniqueId } from 'lodash-es'
 
-export type RadioGroupProps = {
-  options: { label: string; value: string }[]
+export type RadioGroupProps<T extends string | boolean | number = string> = {
```

Comment:
In the "Add Conditions for Use" modal (merged in a previous PR), we use a radio button to select a boolean value (not string). The types needed to be updated for this to be accurately represented.